packages/react-view360/src/index.umd.ts

```diff
@@ -3,5 +3,10 @@
  * egjs projects are licensed under the MIT license
  */
 import View360 from "./View360";
+import * as modules from "@egjs/view360";
+
+for (const name in modules) {
+  (View360 as any)[name] = (modules as any)[name];
+}
```

Comment:
You will have to create index.cjs in react and vue like `@egjs/view360`.

Reply:
I couldn't find `cjs` files in other projects' framework packages.
Why isn't that applied to other components? Like Grid, InfiniteGrid, and others.

Reply:
I asked because it was related to `next.js`.

There will be almost no problems.